app/presenters/publishing_api/landing_page_presenter.rb

```diff
@@ -47,7 +47,11 @@ def document_type
     attr_reader :item
 
     def details
-      YAML.load(item.body, permitted_classes: [Date])
+      body = YAML.load(item.body, permitted_classes: [Date])
+      return body unless body.key?("extends")
+
+      extends = Document.find_by(slug: body.delete("extends")).latest_edition
+      YAML.load(extends.body, permitted_classes: [Date]).merge(body)
```

Comment:
```suggestion
      extends_slug = body["extends"]
      return body if extends_slug.nil?

      extends = Document.find_by(slug: extends_slug).latest_edition
      YAML.load(extends.body, permitted_classes: [Date]).merge(body)
```

Reply:
Just to double-check - this does simplify, but also changes the behaviour so that we publish the extends keyword, is that what you want?

Reply:
That is _not_ what I want, no. I should have looked more closely before assuming you were using `.delete` just to be weird. Sorry 😅 !

I think your implementation is perfectly good in that case. Or maybe with `extends_slug = body.delete("extends")` (so it's got its own line, and it's not hidden inside the `Document.find_by()` call)
```

Reply:
Makes sense - I've moved it out onto its own line.